src/routes/index.tsx

```diff
@@ -4,11 +4,13 @@ import DailyAuthsReport from "../components/daily-auths-report";
 import DailyDropffsReport from "../components/daily-dropoffs-report";
 import ProofingOverTimeReport from "../components/proofing-over-time-report";
 import { Router } from "../router";
-import HomeRoute from "./home-route";
-import createReportRoute from "./report-route";
+import createReportRoute, { ReportRoute } from "./report-route";
 import { Scale } from "../contexts/report-filter-context";
+import Page from "../components/page";
+import Header from "../components/header";
 
-export const ROUTES = {
+const ROUTES: Record<string, ReportRoute> = {
```

Comment:
I think originally the goal was to export `ROUTES` so that we could use it in `bin/generate-routes.js` but that's hard to do because SVG importing seems to get in the way